### PR TITLE
WIP: Live Updates from Xcode Server (proof of concept)

### LINKED ISSUE #7: Hook into the websocket published by xcs to update GitHub status better
There is a websocket published by Xcode Server which posts very detailed updates on the build. You can see it in Xcode when watching a running build. This might be nice to pipe into Builda and maybe, in limited quantities, use for better updating of the state of the build in PRs.

### PR DESCRIPTION
**Pull request by czechboy0**
YES! finally got it working. will need to be moved to XcodeServerSDK, naturally, but this is how you can hook into it. took me months to find this one.

Fixes #7.

### CODE REVIEW

File: Buildasaur/AppDelegate.swift [hunk 1]
```diff
@@ -17,15 +17,55 @@ Also, you can find the log at ~/Library/Application Support/Buildasaur/Builda.lo
...
+                    let id = msg.componentsSeparatedByString(":").first!
+                    self.poll(id)
+                }
+        }
+    }
+    
+    func poll(id: String) {
+        
+        let timestamp = self.timestamp()
+        let url = "https://localhost/xcode/internal/socket.io/1/xhr-polling/\(id)?t=\(timestamp)"
```
Comment (czechboy0) [inline Buildasaur/AppDelegate.swift:60]: Try JSON polling as well.

**Comment by buildasaur** [comment]
Result of Integration **1**
---
*Duration*: 2 minutes and 16 seconds
*Result*: All 19 tests passed, but please **fix 3 warnings**.
*Test Coverage*: 24%.

**Comment by buildasaur** [comment]
## Result of Integration **1**

_Duration_: 2 minutes and 16 seconds
_Result_: All 19 tests passed, but please **fix 3 warnings**.
_Test Coverage_: 24%.

File: Buildasaur/AppDelegate.swift (hunk 1 above)
Comment (czechboy0) [inline Buildasaur/AppDelegate.swift:60]: Actually, that's worse. Sticking with xhr.

**Pull request closed without merging**